sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs

```diff
@@ -192,9 +196,9 @@ public virtual Task<Response<BulkRegistryOperationResponse>> CreateIdentitiesWit
                                 : ExportImportDeviceStatus.Enabled,
                     StatusReason = x.Key.StatusReason,
                     ImportMode = ExportImportDeviceImportMode.Create
-                }.WithTags(x.Value.Tags).WithPropertiesFrom(x.Value.Properties).WithParentScopes(x.Key.ParentScopes));
```

Comment:
what about parent scope? was it removed from swagger?

Reply:
yes it has been removed as it is not GA

Reply:
What do you mean "as it is not GA"?

Reply:
They were adding new properties that are not used in any GA APIs. They are intended to be used by a preview feature which is not released yet.

Reply:
gotcha